packages/shared/src/components/RecommendedTags.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import { ElementPlaceholder } from './ElementPlaceholder';
+import { TagLink } from './TagLinks';
+import type { Tag } from '../graphql/feedSettings';
+
+interface RecommendedTagsProps {
+  isLoading: boolean;
+  tags: Tag[];
+}
+export const RecommendedTags = ({
+  isLoading,
+  tags,
+}: RecommendedTagsProps): ReactElement => {
+  if (isLoading) {
+    return (
+      <div>
+        <ElementPlaceholder className="mb-3 h-4 w-1/5 rounded-12" />
+        <div className="flex gap-2">
+          <ElementPlaceholder className="h-6 w-12 rounded-8" />
+          <ElementPlaceholder className="h-6 w-12 rounded-8" />
+          <ElementPlaceholder className="h-6 w-12 rounded-8" />
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <>
+      {tags.length > 0 && (
```

Comment:
We can probably early return `null` in this case, will be cleaner.

Reply:
Good point, let's do it :)